src/readKongApi.js

```diff
@@ -100,16 +100,8 @@ function parseServiceOrRoutePlugins(plugins, version) {
     return plugins.map(plugin => parsePlugin(plugin, version));
 }
 
-export function parseRoute({ id, created_at, updated_at, service, ...rest }, serviceName = '', config = {}) {
-    let name = null;
-    if (serviceName !== '' && config.services) {
-        const route = (config.services.find((s) => s.name === serviceName).routes || [])
-              .find((r) => r.id === id);
-        if (route) {
-            name = route.name;
-        }
-    }
-    return { name: (name || id), id, attributes: {...rest}, _info: { id, updated_at, created_at } };
+export function parseRoute({ name, id, created_at, updated_at, service, ...attributes }) {
```

Comment:
```suggestion
export function parseRoute({ name, id, created_at, updated_at, ...attributes }) {
```

unused?

Reply:
I think the idea is to explicitly remove `service` from the route. I don't exactly know why, but it seems to work right now

Reply:
Yea we actually just took a look at this. Removing service from the params causes it to be added to attributes and show up on the route that gets returned. I would _think_ Kong can handle having the service field on routes but do you think we should keep this as is for now?

Reply:
My guess is that it's because kongfig expresses the relationship between services and routes as routes being children of services. So any time we're working with routes its in the context of a service to start (and therefore don't need the service reference in the object).

Technically kong allows you to create a route without a service, and I suspect kongfig wouldn't support that. At some point in the future we may want to rework that.

Reply:
@jasalisbury does that cause any errors? Do the integration tests pass still? I'm really fine either way, just wanted to point out that it's not actually a no-op (although it is subtle).

Reply:
@kalilsn It does cause a snapshot error. I'm inclined to leave the param as is to not change too much at once. Thoughts @kalilsn and @mjlumetta?

Reply:
sound logic, i support that